src/index.html

```diff
@@ -11,7 +11,187 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
+    <form action="https://mate-academy-form-lesson.herokuapp.com/create-application"
+      method="post"
+    >
```

Comment:
See checklist item 3
```suggestion
    <form 
      action="https://mate-academy-form-lesson.herokuapp.com/create-application"
      method="post"
    >
```